resources/js/components/Room/MembersComponent.vue

```diff
@@ -106,13 +113,37 @@
         <b-row>
           <b-col cols="12" class="my-1">
             <b-pagination
-              v-if="members.length>settings('pagination_page_size')"
               v-model="currentPage"
               :total-rows="members.length"
               :per-page="settings('pagination_page_size')"
             ></b-pagination>
           </b-col>
         </b-row>
+        <!-- selected rows action buttons -->
+        <b-button-group class="float-md-right" v-if="selectedMembers.length > 0 && members.length > 0">
+            <!-- multiple edit membership role -->
+            <b-button
+              :disabled="isBusy"
+              variant="dark"
+              @click="showEditMultipleMemberModal(selectedMembers)"
+              v-b-tooltip.hover
+              ref="edit-multiple-members-button"
+              :title="$t('rooms.members.multipleEditUser',{numberOfSelectedUsers: selectedMembers.length})"
+            >
+              <i class="fas fa-users-cog"></i>
+            </b-button>
+            <!-- multiple remove member -->
+            <b-button
+              :disabled="isBusy"
+              variant="danger"
+              @click="showRemoveMultipleMemberModal(selectedMembers,{numberOfSelectedUsers: selectedMembers.length})"
```

Comment:
Unused parameter: `{numberOfSelectedUsers: selectedMembers.length}`
```suggestion
              @click="showRemoveMultipleMemberModal(selectedMembers)"
```